src/crypto/crypto-alpha-38.ts

```diff
@@ -29,7 +29,7 @@ export function typedMessageParse(x: string) {
         const json: unknown = JSON.parse(maybeMetadata)
         if (typeof json !== 'object' || json === null || Object.keys(json).length === 0)
             throw new Error('Not a metadata')
-        return makeTypedMessage(end.join('🧩'), new Map(Object.entries(json)))
+        return makeTypedMessageText(end.join('🧩'), new Map(Object.entries(json)))
```

Comment:
```suggestion
        return makeTypedMessageText(end.join('\u{1F9E9}'), new Map(Object.entries(json)))
```

Reply:
Maybe another PR replace all emojis in the codebase. But not this one.